cfgov/v1/tests/atomic_elements/organisms/test_organisms.py

```diff
@@ -183,7 +184,27 @@ def test_tableblock(self):
         self.assertContains(response, 'Header One')
         self.assertContains(response, 'Row 1-1')
         self.assertContains(response, 'Row 2-1')
-        
+
+    def test_tableblock_missing_attributes(self):
+        """Table correctly displays when value dictionary is missing attributes"""
+        value = atomic.table_block.get( 'value', None )
```

Comment:
Minor quibble: it would be better if this test didn't actually modify (using `del`) the value of `atomic.table_block`, because that could potentially impact other tests. Can you copy that value before modifying, e.g.

```py
value = dict(atomic.table_block['value']))
```